modules/js/rackandpinion.js

```diff
@@ -1,680 +1,913 @@
 // module indicator and base variables
 rackPinionModule = true;
-var c = 369
-var rectBase = 600
-var originalWidth1
-var originalWidth2
-var newWidth1
-var newWidth2
-module.verticalSpace = 0
+let c = 369; //connector length
+const rectBase = 600;
+let originalWidth1;
+let originalWidth2;
+var newWidth1;
+var newWidth2;
```

Comment:
Can we change these to let or const?  Most modern JS avoids var if possible.

Reply:
For the variables that I left as var its because when I changed it to let or const the website broke.

Reply:
Ha!  Good to leave them then!  So they must have global scope and be used somewhere else.  Oof.